hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -178,4 +239,97 @@ public static boolean resetCrypto() {
     public static LogLevel getLogLevel() {
         return logLevel;
     }
+
+    /**
+     * Provides the ability to chain put invocations, for example
+     * <code>Hawk.chain().put("foo", 0).put("bar", 1).done()</code>
+     */
+    public static final class HawkChain {
```

Comment:
We can simply use Chain instead of HawkChain. Hawk prefix is not necessary in my opinion.